src/Components/SmartComponents/CVEs/CVEs.cy.js

```diff
@@ -70,17 +70,17 @@ describe('CVE list table', () => {
     it('shows buttons to edit status or business risk', () => {
         cy.get('.ins-c-primary-toolbar__actions > .pf-v5-c-menu-toggle').click();
 
-        cy.get('[data-ouia-component-id="OUIA-Generated-DropdownItem-1"] > .pf-v5-c-menu__item')
+        cy.get('[data-ouia-component-id="edit-business-risk"] > .pf-v5-c-menu__item')
             .contains('Edit business risk')
             .should('exist');
 
-        cy.get('[data-ouia-component-id="OUIA-Generated-DropdownItem-2"] > .pf-v5-c-menu__item')
+        cy.get('[data-ouia-component-id="edit-status"] > .pf-v5-c-menu__item')
             .contains('Edit status')
             .should('exist');
     });
 
-    it.skip('shows row actions', () => {
-        cy.get('[data-ouia-component-id="OUIA-Generated-TableRow-613"] > .pf-v5-c-table__action > .pf-v5-c-menu-toggle')
+    it('shows row actions', () => {
+        cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle')
```

Comment:
There is a chance we have already listed this here https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/src/CypressUtils/selectors.js

Reply:
Thank you, I will keep this in mind when doing another pass of the test improvements.